spot-electron/package.json

```diff
@@ -0,0 +1,36 @@
+{
+    "author": "Jitsi",
+    "build": {
+        "appId": "org.jitsi.spot",
+        "mac": {
+            "category": "public.app-category.business",
+            "target": "default"
+        },
+        "productName": "Jitsi Spot"
```

Comment:
Is it Jitsi Spot or Jitsi-Meet Spot? This repo is called jitsi-meet-spot. We should get verification at some point in the future.

Reply:
Asked Emil, hopefully we get an answer in a few weeks